sdks/java/io/amazon-web-services/src/test/java/org/apache/beam/sdk/io/aws/options/AwsModuleTest.java

```diff
@@ -117,6 +119,17 @@ public void testClasspathPropertiesFileCredentialsProviderSerializationDeseriali
     assertEquals(credentialsFilePath, deserializedCredentialsFilePath);
   }
 
+  @Test
+  public void testSTSCredentialsProviderWrapperSerializationDeserialization() throws Exception {
+    String roleArn = "arn:aws:iam::000111222333:role/TestRole";
+    String roleSessionName = "roleSessionName";
+    STSAssumeRoleSessionCredentialsProvider credentialsProvider =
+        AssumeRoleSessionCredentialsProvider.getInstance(roleArn, roleSessionName)
+            .getSessionCredentialsProvider();
+
+    assertNotNull(credentialsProvider);
```

Comment:
Please, add `Serialization/Deserialization` asserts as we do for other provider tests.